x/programs/cmd/simulator/cmd/root.go

```diff
@@ -148,10 +142,11 @@ func (s *simulator) Init() error {
 
 	// initialize the simulator VM
 	vm := controller.New()
+
 	err = vm.Initialize(
```

Comment:
I think there is a future where we rip this out and just stub the host function to the WASM engine?

Reply:
100% agree. I don't think this is worth doing right at this moment though. I'll create an issue.

Reply:
https://github.com/ava-labs/hypersdk/issues/756